ontologies/vocabs.ttl

```diff
@@ -448,8 +440,10 @@ Adversaries motivated by increased profit or other gains through an unfairly obt
 
 gist:_AttackMotivation_personal-gain
 	a gist:AttackMotivation ;
-	rdfs:label "Personal Gain"^^xsd:string ;
-	skos:altLabel "Personal-Gain"^^xsd:string ;
+	skos:altLabel
+		"Personal Gain"^^xsd:string ,
+		"Personal-Gain"^^xsd:string
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?